vumi_twilio_api/twilio_api.py

```diff
@@ -62,10 +79,28 @@ def format_xml(dct, root=None):
 
     @staticmethod
     def format_json(dct):
-        return json.dumps(dct)
+        if dct.get('Call'):
+            dct = dct['Call']
+        if dct.get('Version'):
+            dct = dct['Version']
```

Comment:
It seems like all resources do this for json. Would you be opposed to doing `_, dct = dct.popitem()`, or would it be better to be explicit like this?